src/intTest/resources/patient/parameters.json

```diff
@@ -62,7 +72,10 @@
           }
         ],
         "managingOrganization": {
-          "reference": "Organization/XX1"
+          "identifier": {
+            "value": "HA456"
```

Comment:
System missing or they don't provide that?

Reply:
It is fine to omit here - this is still our "fake" transaction